specs/eip4844/beacon-chain.md

```diff
@@ -136,6 +139,7 @@ class ExecutionPayloadHeader(Container):
     # Extra payload fields
```

Comment:
also, `excess_blobs` has become `excess_data_gas` with `uint256` type

this should be addressed here and in the `ExecutionPayload`

Reply:
Thanks. Fixed.